src/mscorlib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -1174,6 +1174,25 @@ public virtual void ThrowException(Type excType)
             this.Emit(OpCodes.Throw);
         }
 
+        private static Type GetConsoleType()
+        {
+#if PLATFORM_UNIX
+            Type t = null;
+            try
+            {
+                t = Type.GetType("System.Console, System.Console, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError: false);
```

Comment:
If in the future new versions ship and everything gets bumped to for example `5.0.0.0`, how do you guys now to update this particular line? 
Is that documented somewhere so it can't go wrong or how does that work?